docs/getting_started/notifications-Examples.md

```diff
@@ -0,0 +1,182 @@
+
+
+class WebHookSettings(BaseSettings): # (4)
+    apikey: str
+    teams_webhook_url: Optional[str] = None
+
+
+class SEKOIAIOWebHookAlertAttributes(BaseModel):
+    uuid: str
+    short_id: str
+
+
+class SEKOIAIOWebHookMetadata(BaseModel):
+    version: int = 2
+    created_at: datetime
+    community_uuid: str
+    avatar_uuid: Optional[str] = None
+    user_uuid: Optional[str] = None
+    permissions: Optional[list[str]] = None
+
+
+class SEKOIAIOWebHookEvent(BaseModel): # (5)
+    type: Literal["alert"]
+    action: str
+    attributes: SEKOIAIOWebHookAlertAttributes
+    metadata: SEKOIAIOWebHookMetadata
+
+
+app = FastAPI()
+settings = WebHookSettings()
+
+
+async def push_notification_to_teams(event: SEKOIAIOWebHookEvent):
+    """Simple task that will extract the retrieve the full alert via
+    SEKOIA.IO API and then creates a message that will be posted on
+    Microsoft Teams via a predefined incoming WebHook.
+
+    """
+    async with httpx.AsyncClient() as client:
+        response = await client.get( # (2)
+            f"https://api.sekoia.io/v1/sic/alerts/{event.attributes.uuid}",
+            params={"stix": False},
+            headers={"Authorization": f"Bearer {settings.apikey}"},
+        )
+        if response.status_code == 200:
+            alert = response.json()
+            message = (
+                f"A new alert “{alert['title']}” was created on SEKOIA.IO with "
+                "status “{alert['status']['name']}” and “{alert['urgency']['display']}” severity."
+            )
+            if settings.teams_webhook_url:
+                client.post(settings.teams_webhook_url, json={"text": message}) # (3)
+
+
+@app.post("/hooks/67a7472b-b43c-415d-b1ce-c135fb1dfb0e") # (1)
+async def handle_new_alert(
+    event: SEKOIAIOWebHookEvent, background_tasks: BackgroundTasks
+):
+    background_tasks.add_task(push_notification_to_teams, event)
+```
+
+1. Entrypoint of our HTTP server that receives SEKOIA.IO WebHook notifications. Please define a unique URL, using and UUID for example.
+2. Code in charge of collecting more information on the alert itself by talking to SEKOIA.IO APIs.
+3. Send the final text message to Microsoft Teams.
+4. Environment variables that needs to be defined to let the server work.
+5. Base model that will be used to parse incoming events (this version only handles alerts).
+
+Now, you have to create a new API Key in SEKOIA.IO’s User Center. That one should have at least the right to read alerts (`SIC_READ_ALERT` permissions).
+
+Then you have to install requirements and start the HTTP server with the appropriate environment variables:
+
+```bash
+python -m venv venv
+source venv/bin/activate # (1)
+
+pip install "fastapi[uvicorn]" "uvicorn[classic]" "httpx" # (2)
+
+export APIKEY="eyJ0eXAiOiJ…" # (3)
+export TEAMS_WEBHOOK_URL="https://…"
+
+poetry run uvicorn main:app # (4)
+
+ngrok http 8000 # (5)
+```
+
+1. Create a Python virtual environment to install our code
```

Comment:
@Men-hau same question here, is this a summary?

Reply:
these are the steps describe in the original page to make the actions before using the script